### ISSUE TITLE
Improve wording when "continuing onto" motorways..

### ISSUE THREAD

**Issue by jguthrie100** [graphhopper/graphhopper#1287]
Just a small request, but the language seems a bit strange when driving along a long highway/motorway and being told to continually "continue onto E42" and "Keep left onto E42" for example.

I think "Continue on E42" and "Keep left on E42" would sound much better!

**Comment by karussell** [comment]
Thanks for the recommendation regarding the wording. Regarding the instruction count, this is currently investigated a bit in #1284

**Comment by boldtrn** [comment]
Thanks for the proposal @jguthrie100. You are right, "keep left on" and "continue on" sounds better.

What do you think about "Turn left on main street". Because I think "Turn left onto main street" sounds better?

**Comment by jguthrie100** [comment]
Yep, "turn left onto Main Street" is better!

**Comment by boldtrn** [comment]
Thanks. @karussell how should we approach this issue? 

Should we create a new translation "turn_on". We would then use this translation for keeps and continues? Due to that we might be able to get rid of the "continue_onto" translation. WDYT?

**Comment by karussell** [comment]
For English we might be able to do this simplification but I'm not sure if this is really the same and easy for other languages too. Why not just change the English translation from "onto" into "on"?

**Comment by devemux86** [comment]
Agreed with @karussell to not complicate things so much.
If en translation can be improved, then can simply change that.

**Comment by michaz** [comment]
> Why not just change the English translation from "onto" into "on"?

@karussell 
Because you "turn **onto**" a road, but you "keep left **on**" a road, but the two are currently using the same template "%1$s onto %2$s".

Only "continue onto" has its own entry in the localization table.

**Comment by karussell** [comment]
Ah, ok. Thanks for taking the time to explain that. Makes sense to have it.

**Comment by ethanotran** [comment]
Can I have a go at this issue? Trying to make my first open source contribution.

**Comment by CutestNekoAqua** [comment]
@ethanotran sure! No need to ask! If you wanna do it, just do it :)

**Comment by cheehwatang** [comment]
Hi, I edited the "continue onto %1$s" to "continue on %1$s" in the excel sheet.

However, I faced some issue when I run the script ./core/files/update-translations.sh to update.
It cannot find the txt files.

Error message:
User MINGW64 ~/Documents/Git/graphhopper
$ ./core/files/update-translations.sh
tail: cannot open '' for reading: No such file or directory
tail: cannot open '' for reading: No such file or directory
tail: cannot open '' for reading: No such file or directory
...

Thank you for your help.

**Comment by easbar** [comment]
Yes, you are right the files were moved to `https://github.com/graphhopper/graphhopper-maps`, the new repository where the development of the maps UI takes place now, recently. There is a python script that should do the same as `core/files/update-translations.sh`. I opened this issue so the documentation will reflect this change: https://github.com/graphhopper/graphhopper/issues/2686

**Comment by easbar** [comment]
No, sorry, I was wrong here. Forget about graphhopper-maps for a moment. I think it still works here, you just need to execute the commands as written in the docs: https://github.com/graphhopper/graphhopper/blob/master/docs/core/translations.md#integrate-into-graphhopper

This means you need to pass the excel sheet file to the update-translations.sh script:

```sh
# go to the core folder (no idea why this is needed)
cd core
# download the excel sheet if you haven't already:
curl -L 'https://docs.google.com/spreadsheets/d/10HKSFmxGVEIO92loVQetVmjXT0qpf3EA2jxuQSSYTdU/export?format=tsv&id=10HKSFmxGVEIO92loVQetVmjXT0qpf3EA2jxuQSSYTdU&gid=0' > tmp.tsv
# run the script
files/update-translations.sh tmp.tsv
```

However, you still need to run the python script I mentioned in #2686 to update the translations also for the maps client/UI. Updating the translations here only affects the server response.

**Comment by cheehwatang** [comment]
The method works, thanks. However, the update of translations also replaces some of the other lines, not just the one I changed.
It happens to very other translations. Does this mean this util text files are outdates and not used?

![image](https://user-images.githubusercontent.com/81938708/197969828-c4d06e76-cf43-48e1-8d91-74b02e8b187b.png)

**Comment by easbar** [comment]
No it probably just means there were updates to the spreadsheet other than yours since the last time someone updated the files here in the repo.

**Comment by bennytaccardi** [comment]
Hi, I was wandering if this issue has already been assigned to someone. In case it is not,con you assign it to me? Thank you in advance

**Comment by AzazelSensei** [comment]
Opened https://github.com/graphhopper/graphhopper/pull/3396 for the continue on / onto wording.
